source/testnet/guides/add-baker-desktop.rst

```diff
@@ -0,0 +1,154 @@
+.. _create-baker-desktop:
+
+=========================================
+Add a baker account in the Desktop Wallet
+=========================================
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+A :ref:`baker <baker-concept>` is a node that participates in the network by baking (creating) new blocks that are added to the chain. Each baker has a set of cryptographic keys called baker keys that the node needs to bake blocks. You generate the baker keys in the Desktop Wallet when you add a baker account.
+
+The process of becoming a baker involves the following:
+
+#. Create a set of baker keys.
+#. Register the baker keys with the account.
+#. Start a :ref:`node<run-a-node>` with the baker keys.
+
+When you've completed these steps, the baker node will be able to bake blocks. Read more about :ref:`being a baker <baker-concept>`. This guide goes through all the steps involved in adding a baker in the Desktop Wallet.
+
+Prerequisites
+=============
+There are a couple of things you must set up in preparation before you can start baking.
+
+- Run a node on the Concordium blockchain. Make sure that you have a setup where the node can operate around the clock.
+- Set up a Ledger hardware wallet for your private key.
+- Download and install the Desktop Wallet.
+- Set up an initial account and an identity.
+- Set up a new account that you'll be using as baker account.
+- Verify that the account has a balance of at least ??? GTU.
```

Comment:
The amount of GTU required can be checked with the concordium-client. I'm not sure if we have/should have a separate guide for that.

Reply:
That'll be next release.